po/de.po

```diff
@@ -0,0 +1,347 @@
+msgstr "Neue Notiz"
+
+#: data/ui/help-overlay.ui:37
+msgctxt "shortcut window"
+msgid "Close Note"
+msgstr "Notiz schließen"
+
+#: data/ui/help-overlay.ui:45
+msgctxt "shortcut window"
+msgid "Navigation"
+msgstr "Navigation"
+
+#: data/ui/help-overlay.ui:48
+msgctxt "shortcut window"
+msgid "Show all notes"
+msgstr "Alle Notizen anzeigen"
+
+#: data/ui/help-overlay.ui:54
+msgctxt "shortcut window"
+msgid "Cycle between notes"
+msgstr "Zwischen den Noten wechseln"
+
+#: data/ui/help-overlay.ui:60
+msgctxt "shortcut window"
+msgid "Cycle between notes in reverse order"
+msgstr "Zwischen den Noten in umgekehrter Reihenfolge wechseln"
+
+#: data/ui/help-overlay.ui:68
+msgctxt "shortcut window"
+msgid "Editing"
+msgstr "Bearbeiten"
+
+#: data/ui/help-overlay.ui:71
+msgctxt "shortcut window"
+msgid "Bold"
+msgstr "Fett"
+
+#: data/ui/help-overlay.ui:77
+msgctxt "shortcut window"
+msgid "Italic"
+msgstr "Kursiv"
+
+#: data/ui/help-overlay.ui:83
+msgctxt "shortcut window"
+msgid "Underline"
+msgstr "Unterstreichen"
+
+#: data/ui/help-overlay.ui:89
+msgctxt "shortcut window"
+msgid "Strikethrough"
+msgstr "Durchgestrichen"
+
+#: data/ui/notes.ui:18 data/ui/window.ui:23
+msgid "New Note"
+msgstr "Neue Notiz"
+
+#: data/ui/notes.ui:26 data/ui/window.ui:30
+msgid "Main Menu"
+msgstr "Hauptmenü"
+
+#: data/ui/notes.ui:60
+msgid "Search notes…"
+msgstr "Suche nach Notizen..."
+
+#: data/ui/notes.ui:73
+msgid "No Notes"
+msgstr "Keine Notizen"
+
+#: data/ui/notes.ui:74
+msgid "After you create notes, they will appear here"
+msgstr "Nachdem die Notizen erstellt wurden, werden sie hier angezeigt"
+
+#: data/ui/notes.ui:78
+msgid "_New Note"
+msgstr "_Neue Notiz"
+
+#: data/ui/notes.ui:93
+msgid "No Results"
+msgstr "Keine Ergebnisse"
+
+#: data/ui/notes.ui:94
+msgid "Try a different search"
+msgstr "Versuche eine andere Suche"
+
+#: data/ui/notes.ui:126
+msgid "Keyboard Shortcuts"
+msgstr "Tastenkombinationen"
+
+#: data/ui/notes.ui:130
+msgid "About Sticky Notes"
+msgstr "Über Sticky Notes"
+
+#: data/ui/notes.ui:136
+msgid "Quit"
+msgstr "Beenden"
+
+#: data/ui/theme-selector.ui:21 data/ui/theme-selector.ui:23
+msgid "Follow system style"
+msgstr "Befolgen Sie den Stil des Systems"
+
+#: data/ui/theme-selector.ui:38 data/ui/theme-selector.ui:40
+msgid "Light style"
+msgstr "Heller Stil"
+
+#: data/ui/theme-selector.ui:56 data/ui/theme-selector.ui:58
+msgid "Dark style"
+msgstr "Dunkler Stil"
+
+#: data/ui/window.ui:98
+msgid "Bold"
+msgstr "Fett"
+
+#: data/ui/window.ui:106
+msgid "Italic"
+msgstr "Kursiv"
+
+#: data/ui/window.ui:114
+msgid "Underline"
+msgstr "Unterstreichen"
+
+#: data/ui/window.ui:122
+msgid "Strikethrough"
+msgstr "Durchgestrichen"
+
+#: data/ui/window.ui:138
+msgid "All Notes"
+msgstr "Alle Notizen"
+
+#. TRANSLATORS: eg. 'Translator Name <your.email@domain.com>' or 'Translator Name https://website.example'
+#: src/application.ts:371
+msgid "translator-credits"
+msgstr "alp10711 <93343276+alp10711@users.noreply.github.com>"
```

Comment:
Please use a real email or no email at all

Reply:
Okay, I got it